### DISCUSSION TITLE
Piped.Kavin.Rocks

### DISCUSSION THREAD

**Discussion by Venitryx** [stoatchat/discussions#639] [Feature Suggestions]
Hello,

I want to make a feature request to have embedded videos from Piped.Kavin.Rocks. It's a privacy-focused front-end version of YouTube. I want it so that whenever you paste in a video link from Piped, it appears just as it would for a YouTube video link.

Thank you.

**Comment by DeclanChidlow** [comment]
I don't think this is something Revolt is intending to do right at the moment, but you may benefit from something like LibRedirect.
https://libredirect.github.io/

Do note that this isn't a Revolt endorsed tool and we do not offer support nor any guarantees of it's safe operation.
